lib/streamlit/elements/map.py

```diff
@@ -13,14 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""A Python wrapper around DeckGl."""
 import pandas as pd
-from math import sqrt
-
-from streamlit.logger import get_logger
-import streamlit.elements.deck_gl as deck_gl
-
-LOGGER = get_logger(__name__)
+import json
 
+DEFAULT_MAP = '{"initialViewState":{"latitude":0,"longitude":0,"pitch":0,"zoom":1},"mapStyle":["mapbox://styles/mapbox/light-v10"]}'
+DEFAULT_COLOR = [200, 30, 0, 160]
```

Comment:
Please prefix with `_` (underscore) all private members of this module, like `_DEFAULT_COLOR`